Library/Formula/git-when-merged.rb

```diff
@@ -0,0 +1,13 @@
+class GitWhenMerged < Formula
+  desc "Find the where a commit was merged in git."
+  homepage "https://github.com/mhagger/git-when-merged"
+  url "https://github.com/mhagger/git-when-merged/archive/v1.0.0.tar.gz"
+  sha256 "1be0f2c660e6e0cef66593aac681163b97027a7be028b6d2b3901fee00639ad7"
+  def install
+    bin.install "bin/git-when-merged"
+  end
+
+  test do
+    system "#{bin}/git-when-merged", "-h"
```

Comment:
You're in a sandboxed, clean directory here so you could `git init`, check out a branch, commit something there, merge it into the master and such? Perhaps that'd work.